### PR TITLE
add details about topology

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1305.surge.sh

File: modules/ROOT/pages/clustering/databases.adoc [hunk 1]
```diff
@@ -172,12 +172,15 @@ SHOW SERVERS YIELD serverId, name, address, state, health, hosting;
 In this case, the address for `server01` is `localhost:7687` and thus, the server ID is `8512c9b9-d9e8-48e6-b037-b15b0004ca18`.
 +
 . On one of the servers, use the `system` database and create the database `foo` using the server ID of `server01`.
-If you know how your restored database should be allocated across the cluster, you can also specify the desired topology in the `CREATE DATABASE` command.
+If you know how your restored database should be allocated across the cluster, you can specify the desired topology in the `CREATE DATABASE` command.
+If you don't specify a topology, the database is allocated according to what is specified in the _neo4j.conf_ file.
```
Comment (nick-giles-neo) [inline modules/ROOT/pages/clustering/databases.adoc:176]: The nice bit at the top of this page says:

> If TOPOLOGY is not specified, the database is created according to initial.dbms.default_primaries_count and initial.dbms.default_secondaries_count specified in neo4j.conf. After cluster startup, these values can be overwritten using the dbms.setDefaultAllocationNumbers procedure.

Which covers the two comments I was going to make:
1. Should we specify which conf settings we care about?
2. Should we point out that you can override the conf settings?

Comment (AlexicaWright) [reply modules/ROOT/pages/clustering/databases.adoc:176]: The very next sentence is a link to precisely that section, _See `CREATE DATABASE` for more information._ Is that sufficient?

Comment (nick-giles-neo) [reply modules/ROOT/pages/clustering/databases.adoc:176]: Oh, yeah, fair!

**Review by nick-giles-neo — APPROVED** [review]

**Pull request merged**